pii_recognizer/requirements.txt

```diff
@@ -0,0 +1,10 @@
+mlrun==1.4.0-rc19
```

Comment:
Those requirements are for the tests - so try here also to rely on a stable `mlrun` version